src/history/import_csv_to_db.py

```diff
@@ -1,22 +1,22 @@
 import argparse
-import os
 import sqlite3
+from pathlib import Path
 
 import pandas as pd
 
-BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+BASE_DIR = Path(Path(__file__).resolve()).parent
```

Comment:
Since `Path(__file__).resolve()` is already a `Path`, is there a need to wrap it in a `Path()` constructor again?